### ISSUE TITLE
method `ethClient.SubscribeFilterLogs` got event log twice 

### ISSUE THREAD

**Issue by van-bright** [PlatONnetwork/PlatON-Go#1867]
#### System information

PlatON version: `platon version 1.1.0`
OS & Version:Linux
Commit hash : (if `develop`)

#### Expected behaviour
When target event filtered, there should be noticed only once

#### Actual behaviour
got noticed twice

#### Steps to reproduce the behaviour

1. write some code like below to subscribe event for contract:
```go
package main

import (
    "context"
    "fmt"
    "log"
    "strings"
    "math/big"

    "github.com/PlatONnetwork/PlatON-Go/crypto"
    "github.com/PlatONnetwork/PlatON-Go"
    "github.com/PlatONnetwork/PlatON-Go/common"
    "github.com/PlatONnetwork/PlatON-Go/accounts/abi"
    "github.com/PlatONnetwork/PlatON-Go/core/types"
    "github.com/PlatONnetwork/PlatON-Go/ethclient"
)

func main() {
    client, err := ethclient.Dial("wss://devnetopenapi.platon.network/ws")
    if err != nil {
        log.Fatal(err)
    }

    contractAddress := common.MustBech32ToAddress("lat1xt80enwhxtj2tspr48cv8q9t2sp6shx6klfvmf")
    query := platon.FilterQuery{
        Addresses: []common.Address{contractAddress},
    }

    logs := make(chan types.Log)
    sub, err := client.SubscribeFilterLogs(context.Background(), query, logs)
    if err != nil {
        log.Fatal(err)
    }

    for {
        select {
        case err := <-sub.Err():
            log.Fatal(err)
        case vLog := <-logs:
            fmt.Println(vLog) //  got noticed twice
        }
    }
}
```

2. trigger some events in the contract
3. got `vLog` noticed twice.

**Comment by benbaley** [comment]
fixed in #1868

**Comment by benbaley** [comment]
close by done
